src/components/Tooltip/Tooltip.js

```diff
@@ -416,8 +416,9 @@ export default class Tooltip extends Component {
     );
 
     const triggerClasses = classNames(
-      { [`${prefix}--tooltip__trigger`]: !componentsX },
-      { [`${prefix}--tooltip__label`]: componentsX },
+      // { [`${prefix}--tooltip__trigger`]: !componentsX },
```

Comment:
Did you mean to remove those two lines...?

Reply:
yes I forgot to remove them while modifying the tests, should be fixed now